build.gradle

```diff
@@ -135,6 +135,7 @@ allprojects {
   group = "org.apache.iceberg"
   version = projectVersion
   repositories {
+    maven { url "https://repository.apache.org/content/repositories/orgapachespark-1443/" }
```

Comment:
Thank you! Now, it's on Maven Central. :)

Reply:
Thanks for running the release!